.github/workflows/release.yml

```diff
@@ -12,28 +11,9 @@ permissions:
 
 jobs:
   lint_test:
-    uses: babylonlabs-io/.github/.github/workflows/reusable_node_lint_test.yml@v0.11.0
+    uses: babylonlabs-io/.github/.github/workflows/reusable_node_lint_test.yml@v0.12.0
```

Comment:
Suggest using v0.11.1. You'll have to create the git tag after merging the `.github` repo PR.